pkgs/applications/terminal-emulators/iterm2/default.nix

```diff
@@ -1,48 +1,35 @@
-{ lib, stdenvNoCC, fetchFromGitHub }:
+{ lib, pkgs, stdenv }:
 
  /*
- This derivation is impure: it relies on an Xcode toolchain being installed
- and available in the expected place. The values of sandboxProfile
- are copied pretty directly from the MacVim derivation, which
- is also impure. In order to build you at least need the `sandbox`
- option set to `relaxed` or `false`.
+ This cannot be built from source as it requires entitlements and
+ for that it needs to be code signed. Automatic updates will have
+ to be disabled via preferences instead of at build time.
  */
 
-stdenvNoCC.mkDerivation rec {
+stdenv.mkDerivation rec {
   pname = "iterm2";
-  version = "3.4.14";
+  version = "3.4.15";
+  srcVersion = builtins.replaceStrings ["."] ["_"] version;
 
-  src = fetchFromGitHub {
-    owner = "gnachman";
-    repo = "iTerm2";
-    rev = "v${version}";
-    sha256 = "sha256-sDCnBO7xDpecu2cSjpHwync2DVsj9EKUmgpqEVLtxRM=";
+  src = builtins.fetchurl {
+    url = "https://iterm2.com/downloads/stable/iTerm2-${srcVersion}.zip";
+    sha256 = "0y0vhxwn1cl0y1gjm5fad5zndb4v448mqcksbmmskpgg73h4wn9j";
   };
 
-  patches = [ ./disable_updates.patch ];
-  postPatch = ''
-    sed -i -e 's/CODE_SIGN_IDENTITY = "Developer ID Application"/CODE_SIGN_IDENTITY = ""/g' ./iTerm2.xcodeproj/project.pbxproj
-  '';
+  nativeBuildInputs = with pkgs; [ autoPatchelfHook unzip ];
 
-  preConfigure = "LD=$CC";
-  makeFlagsArray = ["Nix"];
-  installPhase = ''
-    mkdir -p $out/Applications
-    mv Build/Products/Deployment/iTerm2.app $out/Applications/iTerm.app
-  '';
+  dontBuild = true;
```

Comment:
Ok I didn't know that. That means we also remove
```suggestion
```
because the `buildPhase` does not break anything?